src/components/HawkerGrid/HawkerGrid.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Hawker from '../../types/HawkerCentre';
+import { Button, List } from 'semantic-ui-react';
+
+interface Props{
+    hawkerList: Hawker[];
+}
+
+const HawkerGrid : React.FunctionComponent<Props> = (props : Props) => {
+  const{hawkerList} = props;
+  return(
+    <div>
+      {hawkerList.map( x => {
```

Comment:
You should wrap the list around the for loop here. Otherwise, you are just creating multiple lists of one item